ZTP.md

```diff
@@ -88,6 +97,8 @@ Note: DHCP is completely *untrusted* in this flow, we are not going to secure th
 
 Note: DHCP is *not* the only option for discovery. mDNS, SLAAC and others can also be supported as part of sZTP by some devices.
 
+Note: When sZTP is enabled, other DHCP options (except defined in the RFC, code 136,143) are *ignored*.
```

Comment:
well, you still want to configure the IP stack for this client (so you need to consume netmask and router list options at least).

Reply:
correct, I mean ZTP options like 66 & 67...